site/en/blog/progress-in-the-privacy-sandbox-2021-10/index.md

```diff
@@ -0,0 +1,263 @@
+[session](https://watch.videodelivery.net/29bab61e04e8cabf1517e5885c9fe4cf) at
+TPAC which explored an overview of the proposal. There is also a [very early
+prototype version of the
+API](https://github.com/WICG/FedCM/blob/main/explainer/HOWTO.md) available
+behind a flag from Chrome 89, but this is purely for experimentation and will
+change as discussion progresses.
+
+{% Aside %}
+
+The Federated Credentials Management API overlaps with and builds on some of the
+functionality already present in the [Credential Management
+API](https://developer.mozilla.org/en-US/docs/Web/API/Credential_Management_API)._
+
+{% endAside %}
+
+### Cookies
+
+As the cookie-related proposals progress, you should be auditing your own
+`SameSite=None` or **cross-site cookies** and planning the action you will need
+to take on your site.
+
+#### CHIPS
+
+If you set cookies that are sent in cross-site contexts, but in 1:1
+relationships—like iframe embeds, or API calls—you should follow the **[CHIPS
+proposal](https://github.com/WICG/CHIPS)**, or Cookies Having Independent
+Partitioned State. This allows you to mark cookies as "Partitioned" putting them
+in a separate cookie jar per top-level site.
+
+Work is progressing on CHIPS and while the feature is available behind
+`chrome://flags/#partitioned-cookies` and the `--partitioned-cookies` CLI flag,
+it is not yet in a fully testable state. We will provide updated testing and
+debugging details once the implementation is more complete.
+
+{% Img src="image/VWw0b3pM7jdugTkwI6Y81n6f5Yc2/D271bfhBSxVcW29p424S.png",
+alt="The top-level site, green.com has an iframe to red.com. red.com sets a
+cookie with the 'Partitioned' attribute. When the browser is on blue.com with an
+iframe to red.com, no cookie is sent! CHIPS creates a partition per top-level
+site.", width="800", height="240" %}
+
+#### First-Party Sets
+
+If you set cookies for cross-site contexts, but only across sites you own—for
+example, you host a service on your .com that's used by your .co.uk—then you
+should follow **[First-Party
+Sets](https://developer.chrome.com/docs/privacy-sandbox/first-party-sets/)**.
+This proposal defines a way of declaring which sites you want to form a set and
+then marking cookies as "SameParty" so that they are only sent for contexts
+inside of that set.
+
+First-Party Sets are [available for local developer
+testing](https://www.chromium.org/updates/first-party-sets) behind the
+`chrome://flags/#use-first-party-set` and
+`chrome://flags/#sameparty-cookies-considered-first-party` flags, allowing you
+to specify your own set of related sites, and experiment with cookie behavior
+across them.
+
+### Storage Partitioning
+
+The web platform includes other forms of storage that may enable cross-site
+tracking. The TPAC breakout session on **[the state of browser storage
+partitioning](https://docs.google.com/document/d/13oqM9AUnItnDw02zsvpT3DdYYOpIpl0_eTcnbS8rjUY/edit#)**
```

Comment:
```suggestion
partitioning](https://docs.google.com/document/d/13oqM9AUnItnDw02zsvpT3DdYYOpIpl0_eTcnbS8rjUY/preview)**
```

Reply:
The anonymous people can comment and suggest edits, which can get messy. Leaving it as preview doesn't allow them to edit/suggest changes